ips_python/templates/index.html

```diff
@@ -4,29 +4,33 @@
     <h1>
       Find organisations who can help implement your project
```

Comment:
Something that shifts away from search . . . maybe
```suggestion
      Find what you're looking for in IATI descriptions
```